### PR TITLE
Merge day!

### LINKED ISSUE #9899: Merge day! (87/88/89)
Merge day: Monday, March 22nd
Release day: Tuesday, March 23rd

- browser-engine-gecko-beta (87) -> browser-engine-gecko-release (87)
- browser-engine-gecko-nightly (88) -> browser-engine-gecko-beta (88)
- browser-engine-gecko-nightly -> 89

Docs: https://mozac.org/contributing/merge-day

### PR DESCRIPTION
**Pull request by pocmo**
Closes #9899.

### CODE REVIEW

File: components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt [hunk 1]
```diff
@@ -172,7 +172,7 @@ class GeckoEngineSession(
         }
 
         geckoSession.load(loader)
-        Fact(Component.BROWSER_ENGINE_GECKO_BETA, Action.IMPLEMENTATION_DETAIL, "GeckoSession.load").collect()
+        Fact(Component.BROWSER_ENGINE_GECKO_NIGHTLY, Action.IMPLEMENTATION_DETAIL, "GeckoSession.load").collect()
```
Comment (Amejia481) [inline components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt:175]: @pocmo this seem a bit odd if this intentionally ? BROWSER_ENGINE_GECKO_**NIGHTLY**

Comment (pocmo) [reply components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt:175]: Argh, forgot about that. I'll update the PR.

**Review by Amejia481 — APPROVED** [review]
Just [one](https://github.com/mozilla-mobile/android-components/pull/9952/files#r599548294) question apart from that it looks good!

File: components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt (hunk 1 above)
Comment (Amejia481) [reply components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt:175]: 👍🏽

**Pull request merged**